packages/components/src/components/domainproperties/issues/IssuesListDefDesignerPanels.tsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import { List } from 'immutable';
+import produce, { Draft } from 'immer';
+
+import { BaseDomainDesigner, InjectedBaseDomainDesignerProps, withBaseDomainDesigner } from '../BaseDomainDesigner';
+import { getDomainPanelStatus, saveDomain } from '../actions';
+import { resolveErrorMessage } from '../../../util/messaging';
+import DomainForm from '../DomainForm';
+import { DomainDesign } from '../models';
+
+import { IssuesListDefPropertiesPanel } from './IssuesListDefPropertiesPanel';
+import { IssuesListDefModel } from './models';
+
+interface Props {
+    initModel?: IssuesListDefModel;
+    onChange?: (model: IssuesListDefModel) => void;
+    onCancel: () => void;
+    onComplete: (model: IssuesListDefModel) => void;
+    useTheme?: boolean;
+    containerTop?: number; // This sets the top of the sticky header, default is 0
+    successBsStyle?: string;
+    saveBtnText?: string;
+}
+
+interface State {
+    model: IssuesListDefModel;
+}
+
+class IssuesDesignerPanelsImpl extends React.PureComponent<Props & InjectedBaseDomainDesignerProps, State> {
+    constructor(props: Props & InjectedBaseDomainDesignerProps) {
+        super(props);
+
+        this.state = {
+            model: props.initModel || IssuesListDefModel.create({}),
+        };
+    }
+
+    onPropertiesChange = (model: IssuesListDefModel) => {
+        const { onChange } = this.props;
+
+        this.setState(
+            produce((draft: Draft<State>) => {
+                draft.model = model;
+            }),
+            () => {
+                if (onChange) {
+                    onChange(model);
+                }
+            }
+        );
+    };
+
+    onDomainChange = (domain: DomainDesign, dirty: boolean) => {
+        const { onChange } = this.props;
+
+        this.setState(
+            produce((draft: Draft<State>) => {
+                draft.model.domain = domain;
+            }),
+            () => {
+                if (onChange && dirty) {
+                    onChange(this.state.model);
+                }
+            }
+        );
+    };
+
+    onFinish = () => {
+        const { setSubmitting } = this.props;
+        const { model } = this.state;
+        const isValid = IssuesListDefModel.isValid(model);
+
+        this.props.onFinish(isValid, this.saveDomain);
+    };
+
+    saveDomain = () => {
+        const { setSubmitting } = this.props;
+        const { model } = this.state;
+
+        saveDomain(model.domain, model.domainKindName, model.getOptions(), model.name)
+            .then(response => {
+
+                this.setState(
+                    produce((draft: Draft<State>) => {
+                        const updatedModel = draft.model;
+                        updatedModel.exception = undefined;
+                        updatedModel.domain = response;
```

Comment:
I don't think you need the `updatedModel` var here:
```suggestion
                        draft.model.exception = undefined;
                        draft.model.domain = response;
```